mcodingbot/plugins/info.py

```diff
@@ -9,22 +9,20 @@
 
 @plugin.include
 @crescent.command(name="ping", description="Pong!")
-class PingCommand:
-    async def callback(self, ctx: Context) -> None:
-        await ctx.respond(f"Pong! {round(ctx.app.heartbeat_latency*1000)} ms.")
+async def ping(ctx: Context) -> None:
+    await ctx.respond(f"Pong! {round(ctx.app.heartbeat_latency*1000)} ms.")
```

Comment:
```suggestion
    await ctx.respond(f"Pong! {int(ctx.app.heartbeat_latency*1000)} ms.")
```
`round()` has the weird behavior to round to the nearest even, you might not like that
Also, it's personal tastes, but I like when there are spaces around the operator `*` and it follows PEP 8 since it is technically the operator with the lowest priority (I mean, it's alone)

Reply:
Outside scope